build/docker/entrypoint.sh

```diff
@@ -43,7 +43,7 @@ initialize_log_dir() {
 }
 
 copy_provided_plugins() {
-  if [ -d "/opt/plugins" ]; then
+  if ls "/opt/plugins/*.jar" > /dev/null 2>&1; then
```

Comment:
I'm going full-power pedantic mode here: we're checking for _jar_ files, but we copy _everything_. That's probably not right.

pedantic++: I don't like that output redirection. If we don't need to check for jar files specifically (but can trigger the copy for _any_ content, we might be able to get away with something like `[ "$(ls -A /opt/plugins)" ]`